src/pybamm_cookiecutter/entry_point.py

```diff
@@ -37,19 +37,22 @@
 from collections.abc import Mapping
 from typing import Callable
 
-class ParameterSets(Mapping):
+class EntryPoint(Mapping):
     """
-    Dict-like interface for accessing parameter sets through entry points in cookiecutter template.
-    Access via :py:data:`pybamm_cookiecutter.parameter_sets`
+    Dict-like interface for accessing parameter sets and models through entry points in cookiecutter template.
+    Access via :py:data:`pybamm_cookiecutter.parameter_sets` for parameter_sets
+    Access via :py:data:`pybamm_cookiecutter.Model` for Models
 
     Examples
     --------
     Listing available parameter sets:
         >>> import pybamm_cookiecutter
         >>> list(pybamm_cookiecutter.parameter_sets)
         ['Chen2020', ...]
+        >>> list(pybamm_cookiecutter.models)
+        ['SPM', ...]
 
-    Get the docstring for a parameter set:
+    Get the docstring for a parameter set/model:
```

Comment:
This line mentions that one can get a docstring for both a parameter set or a model, so we should provide an additional example similar to the one below.